doc/nrf/protocols/bt/bt_stack_arch.rst

```diff
@@ -87,7 +87,7 @@ Most :ref:`Bluetooth LE samples <ble_samples>` in the |NCS|, including the :ref:
 Exceptions are the :ref:`ble_llpm` sample, which requires the SoftDevice Controller that supports LLPM, and the :ref:`nrf53_audio_app`, which require the SoftDevice Controller that supports :ref:`LE Isochronous Channels <nrfxlib:softdevice_controller_iso>`.
 
 By default, all samples are currently configured to use SoftDevice Controller.
-To use the Zephyr Bluetooth LE Controller instead, set :kconfig:option:`CONFIG_BT_LL_SW_SPLIT` to ``y`` in the :file:`prj.conf` file (see :ref:`configure_application`) and make sure to build from a clean build directory.
+To use the Zephyr Bluetooth LE Controller instead, use the snippet :ref:`zephyr:snippet-bt-ll-sw-split` (see :ref:`app_build_snippets`) and make sure to build from a clean build directory.
```

Comment:
Please point also to anywhere in documentation that Zephyr Controller is experimental or not qualified (unless specified like in case of a particular nRF connect SDK version used for direction finding receivers.)

Reply:
Added a line further up and asked for a review from @emidttun

Reply:
My suggestion: "The Zephyr Bluetooth LE Controller is not supported in |NCS|."

Experimental would indicate that we may support it in the future, but we have no such plans at the moment. Also, see description from line 52. Could be good with a reminder, but should be consistent and not repeat too much.

Reply:
As Erik points out, there's now two different spots on the page that talk about how the Zephyr Controller is not supported. Instead of adding a new note, I'd suggest turning the existing Line 54 to a note, then add a new line after this one with "Nordic Semiconductor does not support the Zephyr Bluetooth LE Controller for production."

Also, the link for snippet renders a full header, not just the snippet name, so the link needs to be edited:
```suggestion
To use the Zephyr Bluetooth LE Controller instead, use the :ref:`bt-ll-sw-split <zephyr:snippet-bt-ll-sw-split>` snippet (see :ref:`app_build_snippets`) and make sure to build from a clean build directory.
```